app/src/pages/common/settingsPage/generalTab/generalTab.jsx

```diff
@@ -111,6 +116,18 @@ export class GeneralTab extends Component {
     });
   }
 
+  componentDidUpdate(prevProps) {
+    if (prevProps.loading !== this.props.loading) {
```

Comment:
After moving loading logic to the plugin the initialization can be done once in the `componentDidMount`.